File: 30 Days of code/C/Day 25/soln.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>
#include <stdbool.h>

bool is_prime(int n) {
    if(n == 1) return false;
    if(n <= 3) return true;

    for(int i = 2; i * i <= n; i++) {
        if(n % i == 0) return false;
    }

    return true;
}

int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    int T, n;
    scanf("%d", &T);

    for(int i = 0; i < T; i++) {
        scanf("%d", &n);
        if(is_prime(n)) {
            printf("Prime\n");
        } else printf("Not prime\n");
    }

    return 0;
}

File: Python/Python If-Else/soln.py
#!/bin/python

# Enter your code here. Read input from STDIN. Print output to STDOUT
N = int(raw_input())
n= N
w = 'Weird'
nw = 'Not Weird'
if n % 2 == 1:
    print(w)
elif n % 2 == 0 and (n>=2 and n<5):
    print(nw)
elif n % 2 == 0 and (n>=6 and n<=20):
    print(w)
elif n % 2 == 0 and (n>20):
    print(nw)   
